src/actions/UserActions.ts

```diff
@@ -44,6 +67,47 @@ export const handleUserLogin = ({ member_code, password }: userAuthData) => {
   });
 };
 
+export const handleUserSignup = async (newUser: userSignupData) => {
+  return new Promise<User>((resolve, reject) => {
+    const startDate = new Date().toISOString().split('T')[0];
+    const url = process.env.NEXT_URL + '/members';
+    const memberCode =
+      `${newUser.first_name[0]}${newUser.last_name[0]}`.toUpperCase() +
+      newUser.dni;
+    const member: userMemberData = {
+      email: newUser.email,
+      password: newUser.password,
+      startDate,
+      namePartner: newUser.first_name,
+      lastnamePartner: newUser.last_name,
+      memberCode,
+      position: 'Socio',
+      type: 'MEMBER',
+      isActive: false,
+    };
+    axios
+      .post(url, member)
+      .then(({ data }) => {
+        Notify.create({
+          message:
+            'Cuenta registrada exitosamente. Usa el siguiente codigo para iniciar sesión' +
+            '<div><strong>' +
+            memberCode +
+            '</strong></div>',
```

Comment:
The message disappears and the user can not see it before it disappears
![image](https://user-images.githubusercontent.com/66505715/208448456-4ca37638-58df-4680-b53d-b4cd08580000.png)

Reply:
Also, change the notification direction to the top of the screen so in mobile phones the message will not be covered by the virtual keyboard